.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 /.fleet
 /.vscode
 dist
+.yarn
```

Comment:
This ignores the `.yarn` directory for this open source repo only, not workspace repos that use yarn v2/3. 

Can you please open an issue so we can discuss the problem first?